File: pickerview/src/main/java/org/jaaksi/pickerview/dialog/IPickerDialog.java
package org.jaaksi.pickerview.dialog;

import android.content.Context;
import org.jaaksi.pickerview.picker.BasePicker;

public interface IPickerDialog {

  /**
   * picker create 时回调
   * @see BasePicker#BasePicker(Context)
   */
  void onCreate(BasePicker picker);

  /**
   * 其实可以不提供这个方法，为了方便在{@link BasePicker#show()}
   */
  void showDialog();
}

File: pickerview/src/main/java/org/jaaksi/pickerview/dialog/IPickerDialog.kt
package org.jaaksi.pickerview.dialog

import org.jaaksi.pickerview.picker.BasePicker

interface IPickerDialog {
    /**
     * picker create 时回调
     * @see BasePicker.BasePicker
     */
    fun onCreate(picker: BasePicker)

    /**
     * 其实可以不提供这个方法，为了方便在[BasePicker.show]
     */
    fun showDialog()
}